### ISSUE TITLE
Use Metadata 

### ISSUE THREAD

**Issue by AustinGMackell** [ImportantCool/gogonzo#11]
At the moment it seems like artefacts are organised on the timeline by date of publication, as in date of upload. If they were organised by the "date-created" from their meta-data. Couldn't that also tell the system whether it's a document or a spreadsheet or an image or a video or audio? One less menu to click through, then.

**Comment by motatoes** [comment]
Clarification needed, are you saying we eliminate the 'artefact type' dropdown and determine the type based on the uploaded file type (MIME type) ?

**Comment by AustinGMackell** [comment]
Yeah that's my suggestion, I think, I don't know what the "MIME type" means. Lik ehow if you click and drag a photo or video into facebook it knows whether it's a video or a photo or whatever and acts accordingly. Also suggesting that the write-line/time-line use the "date created" info from the meta data to organise the artefacts, at the moment it seems to be using the date published/uploaded.
